### ISSUE TITLE
Make sample data in Assign Memory Resources to Containers and Pods topic runtime-neutral

### ISSUE THREAD

**Issue by celestehorgan** [kubernetes/website#31422]
ref: https://github.com/kubernetes/website/issues/30923

Specifically in https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/#exceed-a-container-s-memory-limit, the sample output response specifies a docker container: 

```
lastState:
   terminated:
     containerID: docker://65183c1877aaec2e8427bc95609cc52677a454b56fcb24340dbd22917c23b10f
     exitCode: 137
     finishedAt: 2017-06-20T20:52:19Z
     reason: OOMKilled
     startedAt: null
```

This is a fairly simple remediation: I would do the following:

```
lastState:
   terminated:
     containerID: YOUR_CONTAINER_ID
     exitCode: 137
     finishedAt: 2017-06-20T20:52:19Z
     reason: OOMKilled
     startedAt: null
```

/triage accepted
/good-first-issue

**Comment by KelvinAkpobome** [comment]
> /assign

I made a PR for this already

**Comment by celestehorgan** [comment]
@KelvinAkpobome can you post a link to your PR please, it isn't appearing here in the tracking issue.

**Comment by KelvinAkpobome** [comment]
> @KelvinAkpobome can you post a link to your PR please, it isn't appearing here in the tracking issue.

https://github.com/kubernetes/website/pull/31421

**Comment by nate-double-u** [comment]
closed with #31421

/close
